src/library/tests/test_views.py

```diff
@@ -191,6 +191,10 @@ def test_import_quest_to_current_deck(self):
         # Ensure that the newly imported quest is not visible to students
         self.assertFalse(quest_qs.get().visible_to_students)
 
+        # Ensure that the campaign is NOT imported (it's an orphan quest import)
+        imported_campaign = Category.objects.filter(import_id=campaign.import_id).first()
```

Comment:
A more explicit way to check is using exists() queryset method:

`self.assertFalse(Category.objects.filter(import_id=campaign.import_id).exists())`

https://docs.djangoproject.com/en/5.2/ref/models/querysets/#django.db.models.query.QuerySet.exists